### PR TITLE
[3444][ADD] purchase_deposit

### PR DESCRIPTION
**Pull request by rinaldifirdaus**
[3444](https://www.quartile.co/web#view_type=form&model=project.task&id=3444&active_id=3444&menu_id=)

### CODE REVIEW

**Comment by rinaldifirdaus** [comment]
@yostashiro san, Client approve to install this module to production.
kindly help to merge :)

**Comment by rinaldifirdaus** [comment]
@yostashiro translation added. Kindly your review.

File: purchase_deposit/i18n/ja.po [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+#. module: purchase_deposit
+#: model:ir.model.fields,field_description:purchase_deposit.field_purchase_advance_payment_inv__create_date
+msgid "Created on"
+msgstr "作成日"
+
+#. module: purchase_deposit
+#: model:ir.model.fields,help:purchase_deposit.field_res_company__purchase_deposit_product_id
+#: model:ir.model.fields,help:purchase_deposit.field_res_config_settings__purchase_deposit_product_id
+msgid "Default product used for payment advances."
+msgstr "前払に使用する標準"
```
Comment (yostashiro) [inline purchase_deposit/i18n/ja.po]:
```suggestion
msgstr "前払に使用するデフォルトプロダクト"
```

File: purchase_deposit/i18n/ja.po [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+
+#. module: purchase_deposit
+#: model_terms:ir.ui.view,arch_db:purchase_deposit.res_config_settings_view_form_purchase
+msgid "Deposit Payments"
+msgstr "前払"
+
+#. module: purchase_deposit
+#: model:ir.model.fields.selection,name:purchase_deposit.selection__purchase_advance_payment_inv__advance_payment_method__fixed
+msgid "Deposit payment (fixed amount)"
+msgstr "前払（定額）"
```
Comment (yostashiro) [inline purchase_deposit/i18n/ja.po]:
```suggestion
msgstr "前払(定額)"
```

File: purchase_deposit/i18n/ja.po [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+
+#. module: purchase_deposit
+#: model:ir.model.fields,field_description:purchase_deposit.field_purchase_advance_payment_inv__display_name
+msgid "Display Name"
+msgstr "表示名"
+
+#. module: purchase_deposit
+#: model:ir.model.fields.selection,name:purchase_deposit.selection__purchase_advance_payment_inv__advance_payment_method__percentage
+msgid "Down payment (percentage)"
+msgstr "前払（パーセンテージ）"
```
Comment (yostashiro) [inline purchase_deposit/i18n/ja.po]:
```suggestion
msgstr "前払(%)"
```

File: purchase_deposit/i18n/ja.po [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+
+#. module: purchase_deposit
+#: model:ir.model.fields,field_description:purchase_deposit.field_purchase_advance_payment_inv__id
+msgid "ID"
+msgstr "ID"
+
+#. module: purchase_deposit
+#: model:ir.actions.act_window,name:purchase_deposit.action_view_purchase_advance_payment_inv
+msgid "Invoice Order"
+msgstr "オーダーを請求書"
```
Comment (yostashiro) [inline purchase_deposit/i18n/ja.po]:
```suggestion
msgstr "請求書を作成"
```

Comment (yostashiro) [inline purchase_deposit/i18n/ja.po]:
```suggestion
msgstr "請求書を作成"
```

File: purchase_deposit/i18n/ja.po [hunk 5]
```diff
@@ -0,0 +1,273 @@
...
+
+#. module: purchase_deposit
+#: model_terms:ir.ui.view,arch_db:purchase_deposit.view_purchase_advance_payment_inv
+msgid "Invoice Purchases Order"
+msgstr "購買オーダーを請求書"
```
Comment (yostashiro) [inline purchase_deposit/i18n/ja.po]:
```suggestion
msgstr "請求書を作成"
```

File: purchase_deposit/i18n/ja.po [hunk 6]
```diff
@@ -0,0 +1,273 @@
...
+
+#. module: purchase_deposit
+#: model_terms:ir.ui.view,arch_db:purchase_deposit.view_purchase_advance_payment_inv
+msgid ""
+"Invoices will be created in draft so that you can review them before "
+"validation."
+msgstr ""
+"請求書はドラフトに作成されますので、記帳する前には内容を確認することができます。"
```
Comment (yostashiro) [inline purchase_deposit/i18n/ja.po]:
```suggestion
"請求書はドラフトで作成されますので、記帳前に内容を確認することができます。"
```

File: purchase_deposit/i18n/ja.po [hunk 7]
```diff
@@ -0,0 +1,273 @@
...
+
+#. module: purchase_deposit
+#: model_terms:ir.ui.view,arch_db:purchase_deposit.res_config_settings_view_form_purchase
+msgid "Product used for deposit payments"
+msgstr "前払に使用するプロダクト"
+
+#. module: purchase_deposit
+#: model:ir.model,name:purchase_deposit.model_purchase_advance_payment_inv
+msgid "Purchase Advance Payment Invoice"
+msgstr "購入前払い請求書"
```
Comment (yostashiro) [inline purchase_deposit/i18n/ja.po]:
```suggestion
msgstr "購買前払請求書"
```

File: purchase_deposit/i18n/ja.po [hunk 8]
```diff
@@ -0,0 +1,273 @@
...
+
+#. module: purchase_deposit
+#: model:ir.model.fields,help:purchase_deposit.field_purchase_advance_payment_inv__deposit_taxes_id
+msgid "Taxes used for deposits"
+msgstr "前払に使用する税金"
+
+#. module: purchase_deposit
+#: model:ir.model.fields,help:purchase_deposit.field_purchase_advance_payment_inv__amount
+msgid "The amount to be invoiced in advance, taxes excluded."
+msgstr "事前に請求する金額（税抜き）です。"
```
Comment (yostashiro) [inline purchase_deposit/i18n/ja.po]:
```suggestion
msgstr "前払請求金額（税抜）"
```

File: purchase_deposit/i18n/ja.po [hunk 9]
```diff
@@ -0,0 +1,273 @@
...
+
+#. module: purchase_deposit
+#: model_terms:ir.ui.view,arch_db:purchase_deposit.view_purchase_order_form_inherit
+msgid "Register Deposit"
+msgstr "前払登録"
+
+#. module: purchase_deposit
+#: model:ir.model.fields,help:purchase_deposit.field_purchase_advance_payment_inv__deposit_taxes_id
+msgid "Taxes used for deposits"
+msgstr "前払に使用する税金"
```
Comment (yostashiro) [inline purchase_deposit/i18n/ja.po]:
```suggestion
msgstr "前払に使用する税"
```

File: purchase_deposit/i18n/ja.po [hunk 10]
```diff
@@ -0,0 +1,273 @@
...
+#: model:ir.model.fields,help:purchase_deposit.field_purchase_advance_payment_inv__amount
+msgid "The amount to be invoiced in advance, taxes excluded."
+msgstr "事前に請求する金額（税抜き）です。"
+
+#. module: purchase_deposit
+#. odoo-python
+#: code:addons/purchase_deposit/wizard/purchase_make_invoice_advance.py:0
+#, python-format
+msgid "The percentage of the deposit must be not over 100"
+msgstr "前払のパーセンテージが100を超えないようにしてください"
```
Comment (yostashiro) [inline purchase_deposit/i18n/ja.po]:
```suggestion
msgstr "前払%は100を超えないようにしてください。"
```

File: purchase_deposit/i18n/ja.po [hunk 11]
```diff
@@ -0,0 +1,273 @@
...
+#. module: purchase_deposit
+#. odoo-python
+#: code:addons/purchase_deposit/wizard/purchase_make_invoice_advance.py:0
+#, python-format
+msgid ""
+"The product used to invoice a down payment should be of type \"Service\". "
+"Please use another product or update this product."
+msgstr ""
+"前払請求書で使用するプロダクトはプロダクトタイプが\"サービス\"と設定する必要があります。"
+"前払請求書を作成するには、前払プロダクトのタイプを更新又は他のプロダクトをご利用ください。"
```
Comment (yostashiro) [inline purchase_deposit/i18n/ja.po]:
```suggestion
"前払請求書で使用するプロダクトは「サービス」タイプである必要があります。"
"別のプロダクトを使用するか、このプロダクトのタイプを更新してください。"
```

File: purchase_deposit/i18n/ja.po [hunk 12]
```diff
@@ -0,0 +1,273 @@
...
+#. odoo-python
+#: code:addons/purchase_deposit/wizard/purchase_make_invoice_advance.py:0
+#, python-format
+msgid ""
+"The product used to invoice a down payment should have an invoice policy set"
+" to \"Ordered quantities\". Please update your deposit product to be able to"
+" create a deposit invoice."
+msgstr ""
+"前払請求書で使用するプロダクトは請求方針の設定が\"オーダー数量\"と設定する必要があります。"
+"前払請求書を作成するには、前払プロダクトの請求方針を更新してください。"
```
Comment (yostashiro) [inline purchase_deposit/i18n/ja.po]:
```suggestion
"前払請求書で使用するプロダクトは請求方針が「オーダー数量」である必要があります。"
"前払請求書を作成するには、前払プロダクトの請求方針を更新してください。"
```

File: purchase_deposit/i18n/ja.po [hunk 13]
```diff
@@ -0,0 +1,273 @@
...
+
+#. module: purchase_deposit
+#. odoo-python
+#: code:addons/purchase_deposit/wizard/purchase_make_invoice_advance.py:0
+#, python-format
+msgid ""
+"There is no purchase account defined for this product: %s.\n"
+"You may have to install a chart of account from Accounting app, settings menu."
+msgstr "こちらのプロダクトの購買勘定が設定されていない：%s.\n"
+"会計アプリの設定メニューより、勘定科目をインストールする必要がある場合があります。"
```
Comment (yostashiro) [inline purchase_deposit/i18n/ja.po]:
```suggestion
msgstr "プロダクトに費用勘定が設定されていません：%s\n"
"会計アプリの設定メニューより、勘定科目表をインストールする必要があるかもしれません。"
```

File: purchase_deposit/i18n/ja.po [hunk 14]
```diff
@@ -0,0 +1,273 @@
...
+
+#. module: purchase_deposit
+#: model:ir.model.fields,field_description:purchase_deposit.field_purchase_advance_payment_inv__deposit_taxes_id
+msgid "Vendor Taxes"
+msgstr "仕入先税"
+
+#. module: purchase_deposit
+#: model:ir.model.fields,field_description:purchase_deposit.field_purchase_advance_payment_inv__advance_payment_method
+msgid "What do you want to invoice?"
+msgstr "何を請求したいですか？"
```
Comment (yostashiro) [inline purchase_deposit/i18n/ja.po]:
```suggestion
msgstr "前払請求方針"
```

**Review by yostashiro — CHANGES_REQUESTED** [review]

**Comment by rinaldifirdaus** [comment]
@yostashiro san, translation has been updated. Thank you

**Review by yostashiro — APPROVED** [review]

**Pull request merged**